File: task-3/Go/Sub-Task 2/main.go
package main

import (
    "io"
    "os"
)

func main () {
    r, err := os.Open("input.txt")
    if err != nil {
        panic(err)
    }
    defer r.Close()

    w, err := os.Create("output.txt")
    if err != nil {
        panic(err)
    }
    defer w.Close()
    io.Copy(w, r)
}

File: task-3/Java/Sub-Task 2/Main.java

import java.io.File;
import java.io.FileWriter;
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		File fi = new File("input.txt");
			
		Scanner scanner = new Scanner(fi);
		FileWriter writer = new FileWriter("output.txt");

		while (scanner.hasNextLine())
		{
			String data = scanner.nextLine()+"\n";
			writer.write(data);
		}
			
		scanner.close();
		writer.close();
	}
}
